README.md

```diff
@@ -59,13 +59,51 @@ npm run watch
 ```
 
 ## Tests
+
+### JavaScript
+
 The JavaScript tests currently only cover a small portion of data processing utilities. You can run them with [Node]:
 
 ```sh
 npm install --dev
 npm test
 ```
 
+### Jekyll filters
+We have created a set of [custom Jekyll filters](https://jekyllrb.com/docs/plugins/#liquid-filters) that can be used for templating. The filters in _plugins/eiti_*.rb are tested with [rubydoctest](https://github.com/tslocke/rubydoctest).
+
+#### Testing filters
+
+You can run the unit tests as follows:
+
+```sh
+npm test-ruby
```

Comment:
Ah hah! I got confused, and landed on this completely unrelated page: https://www.npmjs.com/package/ruby